### PR TITLE
New packages to be added

### PR DESCRIPTION
**Pull request by samcom12**

### CODE REVIEW

**Comment by samcom12** [comment]
@alalazo Kindly review

File: var/spack/repos/builtin/packages/py-pymetis/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pymetis/package.py:1]:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-meshpy/package.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-meshpy/package.py:1]: Please add the standard package copyright page.

File: var/spack/repos/builtin/packages/py-pysph/package.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+from spack import *
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pysph/package.py:1]: Please add the standard package copyright page.

File: var/spack/repos/builtin/packages/py-pysph/package.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class PyPysph(Package):
+    """FIXME: Put a proper description of your package here."""
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pysph/package.py:5]: Description is needed here.

File: var/spack/repos/builtin/packages/py-pysph/package.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from spack import *
+
+
+class PyPysph(Package):
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pysph/package.py:4]: Why is this package not inheriting from `PythonPackage`?

File: var/spack/repos/builtin/packages/py-triangle/package.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
+class PyTriangle(PythonPackage):
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-triangle/package.py:1]: Please add the standard package copyright here.

File: var/spack/repos/builtin/packages/py-anuga/package.py [hunk 7]
```diff
@@ -1,8 +1,3 @@
-# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-anuga/package.py:1]: Why was the standard package copyright removed?

**Review by tldahlgren — CHANGES_REQUESTED** [review]
I am just providing some basic comments but will defer to the overarching Python package owner (@adamjstewart) and or @alalazo for more detailed feedback.

**Comment by adamjstewart** [comment]
This would be much easier for me to review if you could split each package into a separate PR.

File: var/spack/repos/builtin/packages/abinit/package.py [hunk 8]
```diff
@@ -30,6 +30,7 @@ class Abinit(AutotoolsPackage):
     homepage = 'http://www.abinit.org'
     url      = 'https://www.abinit.org/sites/default/files/packages/abinit-8.6.3.tar.gz'
 
+    version('9.4.1', sha256='ec74133ef9d247cb6ad44c205675694919cf36d7f1682a9b85cae8bdecdc22a8')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/abinit/package.py:33]: Confirmed the sha258.

**Comment by sethrj** [comment]
There are a handful of new packages, new versions to existing packages, and changes to existing packages all in this one MR, with no comments about how they're related. To echo @adamjstewart , please separate these out to make them easier to review.

**Comment by samcom12** [comment]
It might be difficult for a reviewer to review it. 
Will reopen new PR for each new package with proper description.

**Pull request closed without merging**